src/wasm-binary.h

```diff
@@ -712,7 +712,7 @@ class WasmBinaryBuilder {
 
   // gets a name in the combined function import+defined function space
   Name getFunctionIndexName(Index i);
-  void getResizableLimits(Address& initial, Address& max, Address defaultIfNoMax);
+  void getResizableLimits(Address& initial, Address& max, bool& shared, Address defaultIfNoMax);
```

Comment:
The first 3 parameters are out-parameters, and the last one is an in-parameter, so I wanted to keep them together.